tools/testsys/src/aws_resources.rs

```diff
@@ -191,6 +180,18 @@ pub(crate) async fn ec2_crd(
                 .testsys_agent_pull_secret
                 .clone(),
         )
+        .device_mappings({
```

Comment:
`device_mappings` is the field of EC2Config struct [here](https://github.com/bottlerocket-os/bottlerocket-test-system/blob/56aecbaf0d2c6229d035fc02af7ce5701abdc940/bottlerocket/types/src/agent_config.rs#L298), so I have to use `device_mappings` instead of `set_device_mappings`.

```
ec2_builder
145 | |         .node_ami(bottlerocket_input.image_id)
146 | |         .instance_count(2)
147 | |         .instance_types::<Vec<String>>(
...   |
183 | |         .set_device_mappings({
    | |_________-^^^^^^^^^^^^^^^^^^^
    |
help: there is a method `device_mappings` with a similar name
    |
183 -         .set_device_mappings({
183 +         .device_mappings({
```